src/App/App.tsx

```diff
@@ -43,14 +50,13 @@ export default function App() {
         isMovieInfoOpen={isMovieInfoOpen}
         movie={movie}
       />
-      <Main
-        onMovieModalOpen={handleMovieModalOpen}
-        onDeleteModalOpen={handleDeleteModalOpen}
-        onMovieInfoOpen={handleMovieInfoOpen}
-        isLoading={isLoading}
-        movies={movies}
-        loadingError={error}
-      />
+      <CardContext.Provider value={cardContextValue}>
```

Comment:
I don't think that CardContext is a good name, because this context is responsible for modals.

```
<ModalProider>
// application markup
</ModalProvider>
```

Reply:
Changed it to AppContext and placed it in the ModalProvider component.